validation/src/test/scala/hmda/validation/rules/lar/validity/V455Spec.scala

```diff
@@ -0,0 +1,39 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V455Spec extends LarEditCheckSpec {
+  import Gen.posNum
+
+  property("Succeeds when Applicant Ethnicity is not 1, 2, or 3") {
```

Comment:
The only possible ethnicity values are 1,2,3,4. This will generate invalid values, not critical for this test but this could be changed to generate an ethnicity of 4 to be more precise